frontend/src/pages/order/FulfilPurchaseRequestPage.tsx

```diff
@@ -0,0 +1,84 @@
+import { useQuery } from "@tanstack/react-query";
+import BreadCrumb from "../../components/event/Breadcrumb";
+import EventBanner from "../../components/event/card/EventBanner";
+import { getEvent } from "../../axios/event/event";
+import { useParams } from "react-router-dom";
+import Heading from "../../components/common/headings/Heading";
+import { Box } from "@mui/material";
+import FulfilPurchaseRequestTable from "../../components/event/table/FulfilPurchaseRequestTable";
+import { Activity } from "../../types/activity";
+
+const FulfilPurchaseRequestPage = () => {
+    const { id } = useParams();
+
+    const { data: event } = useQuery({
+        queryKey: ["event", id],
+        queryFn: () => getEvent(0),
+    });
+
+    const activities: Activity[] = [
```

Comment:
Try logging the output of `event` from the query above, notice any particular field inside that will be useful for rendering `activities`?